### DISCUSSION TITLE
Realtime Issue in flutter web only in ios device browser.

### DISCUSSION THREAD

**Discussion by sabikrahat** [pocketbase/pocketbase#2981] [Q&A]
I'm using pocketbase latest version in my flutter project where I need realtime data. I used `fetch_client` package to use realtime service in flutter web...... it is working fine in macos browser, windows laptop browser and also in android mobile browser..... but showing error in ios device while I try to enter the website using safari or chrome browser. I have hosted pocketbase using fly.io. I have inspect the error behind this and attching the oputputs here. If you anyone know how to solve this please help.

![image](https://github.com/pocketbase/pocketbase/assets/49031412/5faa846e-297d-429c-a09e-a823e7c6aa40)

![image](https://github.com/pocketbase/pocketbase/assets/49031412/76376473-e648-4b22-bff5-17a9eb8ff1b7)

**Comment by ganigeorgiev** [comment]
Unfortunately the above screenshots don't show much so that it can be investigated.

If the above errors are fired on the same interval (eg. 60sec) then this is most likely because fly.io has 60sec timeout interval for idle connections (_see https://github.com/pocketbase/pocketbase/discussions/642_) and the errors could be safely ignored.

**Reply by sabikrahat** [reply]
No it's not the same.. I'm getting error at my splash screen..... where I'm getting user data in real time. Here I'm giving you some more screenshots.

![image](https://github.com/pocketbase/pocketbase/assets/49031412/eb1bcf74-9e7f-42f1-b5ee-e8fde39c8164)

![image](https://github.com/pocketbase/pocketbase/assets/49031412/7879d1ec-4b56-4c53-887a-95e4ab293917)

**Reply by sabikrahat** [reply]
@ganigeorgiev if you need any other details please let me know. I think the issue is fetching data in real-time on ios devices. I'm not sure fully.

**Reply by ganigeorgiev** [reply]
Sorry, I don't think I can help you with this.
